docs/general/dns-filtering-syntax.md

```diff
@@ -259,6 +259,8 @@ The `dnsrewrite` response modifier allows replacing the content of the response
 
 **Rules with the `dnsrewrite` response modifier have higher priority than other rules in AdGuard Home.**
 
+Responses to all requests for a host matching a `dnsrewrite` rule shall be replaced. The replacement response's answer section shall only contain RRs that match the request's query type and, possibly, CNAME RRs. Note that this means that responses to some requests may become empty if the host matches a `dnsrewrite` rule.
```

Comment:
Perhaps, clarify that “empty” here means NODATA?

Reply:
@ngorskikh Could it be not "shall" but "will"? Since this is at first information for users, not only for implementation developers.